compiler-rt/lib/builtins/cpu_model/aarch64.c

```diff
@@ -73,7 +73,7 @@ struct {
 #elif defined(__ANDROID__)
 #include "aarch64/fmv/mrs.inc"
 #include "aarch64/fmv/android.inc"
-#elif __has_include(<sys/auxv.h>)
+#elif defined(__linux__) && __has_include(<sys/auxv.h>)
 #include "aarch64/fmv/mrs.inc"
 #include "aarch64/fmv/getauxval.inc"
 #else
```

Comment:
Is the expectation that we hit the unimplemented case on OpenBSD?

Reply:
Correct.